dl1_data_handler/dl1_data_writer.py

```diff
@@ -0,0 +1,798 @@
+# -*- coding: utf-8 -*-
+"""Load data from ctapipe EventSources and dump to file."""
+
+from abc import ABC, abstractmethod
+import pkg_resources
+import argparse
+import logging
+import os
+import re
+import multiprocessing
+import warnings
+import sys
+
+import yaml
+import numpy as np
+import tables
+from ctapipe import io, calib
+
+import table_definitions as table_defs
```

Comment:
Here you want to do `from . import table_definitions as table_defs` I think

Reply:
I think this change would have to be coupled with the other point you raised. If the script is going to be moved to a separate file, outside of the package, then the import should definitely be absolute (like it is now), not relative. If it stays where it is inside the package, I think either way would work fine.